ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2410,34 +2446,39 @@ private enum UnityLayers {
              Log.Fatal("missing Cartesian initial state for " + body.name);
           }
           int? parent_index = body.orbit?.referenceBody.flightGlobalsIndex;
+          // GetUniqueValue resp. GetAtMostOneValue corresponding to required
+          // resp. optional in principia.serialization.GravityModel.Body.
```

Comment:
Maybe this intent (required vs. optional == unique vs. atmost) should be documented in `config_node_extensions.cs`?

Reply:
Done. Not *quite* `==`, since in some circumstances we make `required` fields optional by defaulting to the parameters of the game.